inference-engine/tests/functional/inference_engine/serialization/single_layer/activation.cpp

```diff
@@ -57,7 +57,6 @@ const std::map<ActivationTypes, std::vector<std::vector<float>>> activationTypes
         {HSigmoid,              {}},
         {RoundHalfToEven,       {}},
         {RoundHalfAwayFromZero, {}},
-        {Erf,                   {}},
```

Comment:
Why did you remove `Erf` serialization tests?

Reply:
It is duplicated . See the line 50 in the file.